Chapter-3/chapter-3.md

```diff
@@ -0,0 +1,17 @@
+# Chapter 3
+
+* Some people think that human psychology naturally works better when it receives information in the form a person in a natural environment would receive it.
+
+* Working with samples transforms a problem in calculus into a problem in data summary, into a frequency format problem.
+
+* We can sample the posterior distribution in order to summarize it:
+
+    * Intervals of defined boundaries: We get the accumulated posterior probabilities within those boundaries
+
+    * Intervals of defined mass: We get the quantiles that accumulate that mass of probability. There are an infinite number of intervals that meet those requirements. We can define, for example, the Highest posterior density interval, which is the narrowest interval containing the specified probability mass.
+
+
+    * Point estimates: Given the posterior distribution of a parameter, what value should we report? It’s very common to report the parameter value with the highest posterior probability. (Maxima a posteriori or MAP). Another option is to choose a loss function that associates a parameter value with a cost and report the value that minimizes such cost. The key insight here is that different loss functions imply different point estimates. The two most common examples are the absolute loss |d-p|, which leads to the median as the point estimate, and the quadratic loss (d-p)², which leads to the posterior mean as the point estimate. Also, depending on the problem, you could define an unique loss function.
```

Comment:
`you could define an unique loss function.`
`you could define **a** unique loss function.`